engine_details/development/debugging/profiling/tracing_profilers.rst

```diff
@@ -0,0 +1,235 @@
+.. _doc_tracing_profilers:
+
+Tracing Profilers
+=================
+
+Godot currently supports two tracing profilers:
+`Tracy <https://github.com/wolfpld/tracy>`__ and `Perfetto <https://perfetto.dev>`__.
+
+In order to use either of them, you'll need to build the engine from source.
+If you've never done this before, please read
+:ref:`these docs <doc_compiling_index>` for the platform you want to profile on.
+You'll need to perform the same steps here, but with some additional arguments
+for ``scons``.
+
+.. _doc_tracy_profiler:
+
+Tracy for Windows, Linux, and macOS
+-----------------------------------
+
+Tracy is an Open Source profiler that runs on a wide variety of platforms,
+including Windows, Linux, and macOS. While it is primarily a tracing profiler,
+it can also periodically sample data like a
+:ref:`sampling profiler <doc_sampling_profilers>`, giving some of the benefits
+of both approaches.
+
+Build Godot with Tracy support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, clone the latest version of the Tracy source code ("0.13.0" at the
+time of writing) using Git:
+
+.. code-block:: shell
+
+    git clone -b v0.13.0 --single-branch https://github.com/wolfpld/tracy.git
```

Comment:
Tracy has relatively frequent releases so this will get out of date quickly, and we don't want to have to update it regularly (which would add churn and also invalidate translations), so I'd suggest looking for an alternative that doesn't hardcode a version number.

You can use https://github.com/wolfpld/tracy/releases/latest to link to the latest tag, but that's not usable for the git clone command.